Persistent logs initial documentation

daisyfaithauma opened the pull request:
- This is initial persistent logs documentation

kathayl commented:
Can we
1) change the page name to "Logging"
2) when this page goes live, we will also need to update the existing "analytics and logging" page to just be "Analytics", add the cool new analytics section you wrote, and remove the logging section

daisyfaithauma commented:
> Can we
> 
> 1. change the page name to "Logging" (matches how Workers does it)
> 2. when this page goes live, we will also need to update the existing "analytics and logging" page to just be "Analytics", add the cool new analytics section you wrote, and remove the logging section
> 3. the font of the header section is weird. can it match how we have header instructions in the caching page?
> 
> <img alt="image" width="804" src="https://private-user-images.githubusercontent.com/153706637/367848051-7912e9b4-76b1-4f75-bc84-d3d67f0a7d3a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NDQ3NjgsIm5iZiI6MTcyNjg0NDQ2OCwicGF0aCI6Ii8xNTM3MDY2MzcvMzY3ODQ4MDUxLTc5MTJlOWI0LTc2YjEtNGY3NS1iYzg0LWQzZDY3ZjBhN2QzYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMFQxNTAxMDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZTE0ZDU4ZWQwNTM5NWM5Y2FjY2I0MWRlNDAyODdiYmU4NDZhZmY5ZjRmZTcwOGJmYzFhYWI0NzI4ZmM1ZGQwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z5wpOS91uW-A0h2_IYhCUW0Ihci7qWTRGhQiNJv1nXw"> 4) add to changelog 5) reminder - we still have to add in the logpush section to this draft PR

- Renamed page to logging
- Fixed header
-

kathayl commented on src/content/docs/ai-gateway/observability/logging.mdx:
```diff
@@ -0,0 +1,58 @@
+
+Your AI Gateway dashboard shows logs of individual requests, including the user prompt, model response, provider, timestamp, request status, token usage, cost, and duration. These logs persist, giving you the flexibility to store them for your preferred duration and do more with valuable request data. You can store up to 10 million logs per gateway.
+
+When you are nearing your plan limit, an alert will be displayed in the dashboard. If your limit is reached, new logs will stop being saved. To continue saving logs, you must delete older logs to free up space for new logs.
```
can we link "limit" to the limits page

Oxyjun commented on src/content/docs/ai-gateway/observability/logging/index.mdx:
```diff
@@ -0,0 +1,56 @@
+
+You can store up to 10 million logs per gateway. If your limit is reached, new logs will stop being saved. To continue saving logs, you must delete older logs to free up space for new logs.
+
+To learn more about your plan limits, refer to the [Pricing](/ai-gateway/pricing/) page.
```
```suggestion
To learn more about your plan limits, refer to [Pricing](/ai-gateway/pricing/).
```

Oxyjun commented on src/content/docs/ai-gateway/observability/logging/index.mdx:
```diff
@@ -0,0 +1,56 @@
+
+<Render file="logging" />
+
+:::note[Note]
```
```suggestion
:::note
```

Oxyjun commented on src/content/docs/ai-gateway/observability/logging/index.mdx:
```diff
@@ -0,0 +1,56 @@
+
+To override the default logging behavior set in the settings tab, you can define headers on a per-request basis.
+
+## Collect logs (cf-aig-collect-log)
```
```suggestion
## Collect logs (`cf-aig-collect-log`)
```

Oxyjun commented on src/content/docs/ai-gateway/observability/logging/index.mdx:
```diff
@@ -0,0 +1,56 @@
+
+The `cf-aig-collect-log` header allows you to bypass the default log setting for the gateway. If the gateway is configured to save logs, the header will exclude the log for that specific request. Conversely, if logging is disabled at the gateway level, this header will save the log for that request.
+
+In this example, the gateway is configured to save logs, however we do not want this specific request saved.
```
```suggestion
In the example below, we use `cf-aig-collect-log` to bypass the default setting to avoid saving the log.
```
Suggestion, feel free to ignore :D

daisyfaithauma commented:
Re done on another PR
